modules/exploits/unix/dhcp/rhel_dhcp_client_command_injection.rb

```diff
@@ -0,0 +1,76 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'rex/proto/dhcp'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::DHCPServer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'DHCP Client Command Injection (DynoRoot)',
+      'Description'    => %q|
```

Comment:
I believe curl braces are the standard for msf modules.

`%q{  }`

Reply:
Changed / fixed